slurm/install/installlib.py

```diff
@@ -1,16 +1,25 @@
+import base64
 import grp
+from hashlib import md5
 import json
 import logging
 import os
 import pwd
 import re
 import shutil
+from ssl import SSLContext
+import ssl
 import subprocess
 import tempfile
-import time
+from time import sleep as _sleep
```

Comment:
Just trying to understand the rationale behind doing this? It seems we are just replacing time.sleep() in code with sleep() but is it serving some other purpose as well?

Reply:
See related comment